### PR TITLE
Add io.github.dimtpap.coppwr

### PR DESCRIPTION
**Pull request by dimtpap**
## Replaces https://github.com/flathub/xyz.dimtpap.coppwr
https://github.com/flathub/xyz.dimtpap.coppwr/pull/5

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by dimtpap** [comment]
bot, build io.github.dimtpap.coppwr

**Comment by hfiguiere** [comment]
bot, build io.github.dimtpap.coppwr

File: io.github.dimtpap.coppwr.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			],
+			"cleanup": [
+				"/bin",
+				"/include"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.freedesktop.org/pipewire/pipewire.git",
+					"tag": "0.3.77",
```
[the remark this answers is not in the document]
Comment (dimtpap) [reply io.github.dimtpap.coppwr.json:59]: The runtime provided PipeWire is built without some modules like the profiler one that this app uses

[the remark this answers is not in the document]
Comment (hfiguiere) [reply io.github.dimtpap.coppwr.json:59]: and how about the version?

can't you just build the module to match?

[the remark this answers is not in the document]
Comment (hfiguiere) [reply io.github.dimtpap.coppwr.json]: current runtime version is 0.3.83

**Comment by dimtpap** [comment]
bot, build io.github.dimtpap.coppwr

**Pull request closed without merging**